src/gravel/controllers/orch/system.py

```diff
@@ -32,9 +43,23 @@ def set_address(addr: str) -> bool:
     try:
         subprocess.check_output(restart_cmd, stderr=subprocess.PIPE)
     except subprocess.CalledProcessError as e:
-        raise ChronyRestartError("Unable to restart chronyd.service") from e
+        raise SystemCtlError("Failed to restart unit 'chronyd.service'.") from e
```

Comment:
Hmmm... I know it's nitpicking and I'm not sure how others think about it... but I'm in favor of keeping at least the word "Unable". Just because of consistency reasons. It's already used in most of the error messages.

Another option would be to rephrase all error messages from "Unable" to "Failed".

Reply:
I'll change it back to `Unable`.